tesseract_environment/src/core/environment.cpp

```diff
@@ -1316,4 +881,492 @@ Environment::Ptr Environment::clone() const
 
   return cloned_env;
 }
+
+//////////////////////////////////////////////////////////////
+/// External Helper Commands wrapping environment commands ///
+//////////////////////////////////////////////////////////////
+
+bool Environment::addKinematicsInformation(const tesseract_scene_graph::KinematicsInformation& kin_info)
+{
+  return applyCommand(std::make_shared<AddKinematicsInformationCommand>(kin_info));
+}
+
+bool Environment::addLink(const tesseract_scene_graph::Link::ConstPtr& link)
+{
+  return applyCommand(std::make_shared<AddCommand>(link));
+}
+
+bool Environment::addLink(const tesseract_scene_graph::Link::ConstPtr& link,
+                          const tesseract_scene_graph::Joint::ConstPtr& joint)
+{
+  return applyCommand(std::make_shared<AddCommand>(link, joint));
+}
+
+bool Environment::moveLink(const tesseract_scene_graph::Joint::ConstPtr& joint)
+{
+  return applyCommand(std::make_shared<MoveLinkCommand>(joint));
+}
+
+bool Environment::addLink(tesseract_scene_graph::Link link)
+{
+  return applyCommand(std::make_shared<AddCommand>(std::make_shared<tesseract_scene_graph::Link>(std::move(link))));
+}
```

Comment:
This naming disconnect seems odd. If we are going to break it, now seems like the time.

Reply:
I agree. I will make this change in a separate command after the clean up.

Reply:
A new commit has been added making this change.